kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -52,6 +65,10 @@ define([
             this.fileStagingClient = new FileStaging(
                 this.ftpUrl, this.userId, {token: runtime.authToken()});
 
+            this.genericClient = new GenericClient(Config.url('service_wizard'), {
+              token: Runtime.make().authToken()
```

Comment:
There's already an initialized local handle on Runtime. So just `runtime.authToken()` works here.

Reply:
All changes made.
* I got rid of references to the FileStaging() client in favor of StagingServiceClient
* It can't use the ftp url, since it doesn't have the same routes. I added a staging_api_url to src/config.json, including the fake pointers for appdev, next, and prod services which don't exist yet.

Reply:
the FTP url is, i guess, deprecated at this point. we can remove it later. I'm planning on scouring through the repo and ditching all kinds of crap - that can be part of it. :)